static/app/views/replays/detail/memoryChart.tsx

```diff
@@ -294,8 +297,71 @@ function MemoryChartContainer({
   }, [currentHoverTime, startTimestampMs, theme]);
 
   return (
-    <MemoizedMemoryChart ref={chart} startTimestampMs={startTimestampMs} {...props} />
+    <Fragment>
+      <Global styles={getChartTooltipStyles(theme)} />
```

Comment:
would it be beneficial if the charting library did this itself?
Then when someone else uses `appendToBody: true` it a) just works b) reduces copy+pasted global styles